bins/packages/rfs/rfs.sh

```diff
@@ -0,0 +1,30 @@
+RFS_VERSION="2.0.0"
+RFS_CHECKSUM="d6e1c3036d8b0d7d6df4d67fe17a0502"
+RFS_LINK="https://github.com/threefoldtech/rfs/releases/download/v${RFS_VERSION}/rfs"
+
+download_rfs() {
+    download_file ${RFS_LINK} ${RFS_CHECKSUM} rfs-${RFS_VERSION}
+}
+
+prepare_rfs() {
+    echo "[+] prepare 0-fs"
+    github_name "0-fs-${RFS_VERSION}"
```

Comment:
What does github_name do? can we make the name `rfs` instead of `0-fs`?  Also in the echo

Reply:
followed up in https://github.com/threefoldtech/zos/issues/2075